src/main/java/me/totalfreedom/totalfreedommod/command/Command_vanish.java

```diff
@@ -15,7 +15,7 @@
 import org.bukkit.scheduler.BukkitRunnable;
 
 @CommandPermissions(level = Rank.ADMIN, source = SourceType.ONLY_IN_GAME)
-@CommandParameters(description = "Vanish/unvanish yourself.", usage = "/<command> [-s[ilent]]", aliases = "v")
+@CommandParameters(description = "Vanish/unvanish yourself.", usage = "/<command> [-q]", aliases = "v")
```

Comment:
I thought we were using -s? Or have I got muddled up given this is the 107th or so file I've looked at...?

Reply:
nope, we are using -q